tools/plugins/com.liferay.ide.server.tomcat.ui/src/com/liferay/ide/server/tomcat/ui/editor/LiferayServerSettingsEditorSection.java

```diff
@@ -962,6 +963,19 @@ else if (path.equals(installDirPath))
 
 			}
 			*/
+            String memoryValue = memoryArgs.getText();
+            String[] memory = DebugPlugin.parseArguments( memoryValue );
+            if( !CoreUtil.isNullOrEmpty( memoryValue ) )
+            {
+                for( String str : memory )
+                {
+                    if( !( str.startsWith( "-X" ) ) ) //$NON-NLS-1$
+                    {
+                        return new IStatus[] { new Status(
+                            IStatus.ERROR, LiferayServerUI.PLUGIN_ID, "error memeory format" ) }; //$NON-NLS-1$
```

Comment:
Error msg should be
"Error in memory argument format, expecting it to start with "-X""